### ISSUE TITLE
Expand usage check

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBO-Dashboard#46]
From operations committee meeting on 18th May 2021:

- we should tighten the usage check to look for URLs that show the usage (some kind of proof). URL should be resolvable.
- We should relax the severity to WARN for the dashboard so that new ontologies without external usage can still pass the dashboard.

**Comment by nataled** [comment]
Stated in meeting, but just to document here: This is another example of a principle that is intended to apply to established ontologies seeking review and inclusion as 'OBO Foundry' as opposed to OBO Library (yes, that again!).

Soapbox: The dashboard itself was developed for this purpose, not to gatekeep new ontologies. While it surely helps in terms of guidance for improvement for a new ontology (and old too!), we shouldn't be using it for an unintended purpose. Perhaps there could be a 'switch' put in place to allow a specific subset of tests for new ontologies?

**Comment by matentzn** [comment]
@nataled I was under the impression that we did elevate the Dashboard to be a gatekeeper in the previous operations call? We just merged [this](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1497/files) after all - during the committee call!

However, I do think that if we encounter corner cases where we need to be more permissive, we will relax the checks for the new ontology request dashboard. Would that be ok?

**Comment by nataled** [comment]
@matentzn yes, that's correct (about the gatekeeper aspect), but obviously it wasn't done with full consideration of the impact on a principle by principle basis. Rather, the notion of using the dashboard as gatekeeper was approved in concept. Now we need to deal with the particulars. Bear in mind that I don't have objections to using the dashboard as gatekeeper--I think it's a good idea, actually--but we need to be mindful of what the principles state vs what the dashboard is assessing.

I'm not too keen on relaxing dashboard checks on all ontologies to satisfy its new use for new ontologies, which is why I suggested a switch for that purpose. I see two possible ways to approach this with respect to what the switch does: (1) relax the requirement as you say, or (2) skip the check altogether. I don't feel strongly either way, though I should point out that if the standard for acceptance is a green-across-the-board requirement, this will be easier using (2). Using mechanism (1), the standard for admission could be 'non-red across the board'.
